tests/unit/services/apigateway/test_handler_request.py

```diff
@@ -96,26 +96,13 @@ def test_parse_request(self, dummy_deployment, parse_handler_chain, get_invocati
         assert context.region == TEST_AWS_REGION_NAME
 
         assert context.invocation_request["http_method"] == "GET"
-        assert context.invocation_request["raw_headers"] == Headers(
+        assert context.invocation_request["headers"] == Headers(
             {
                 "host": host_header,
                 "test-header": "value1",
                 "test-header-multi": ["value2", "value3"],
-                "content-length": len(body),
```

Comment:
It is now removed and no longer accessible in the invocation requests. This is on parity with aws.